docs/kratos/self-service/flows/account-recovery-password-reset.mdx

```diff
@@ -0,0 +1,205 @@
+     # Defines how long a recovery code is valid for (default 1h)
+     lifespan: 15m
+
+flows:
+ recovery:
+   enabled: true
+
+   # Defines how long a recovery flow (the UI interaction) is valid for (default 1h)
+   lifespan: 15m
+   use: code # Define which strategy to use, either `code` or `link`
+```
+
+Keep in mind, that both methods require sending out emails to your users, so an SMTP server is required. Read more
+[here](../../emails-sms/sending-emails-smtp).
+
+</TabItem>
+</Tabs>
+````
+
+## Account recovery addresses
+
+To start account recovery, Ory Identities (Ory Kratos) needs to know which address to send the recovery message to. In most cases
+this is the address the user provided when registering their account. Other fields inside the `traits` section are supported as
+well.
+
+:::note
+
+If the email address used for recovery is the same as the email used for verification, but the account hasn't been activated yet
+(see [Account Activation](./verify-email-account-activation.mdx)), the address will also be verified by completing the account
+recovery successfully.
+
+:::
+
+To specify a trait of the identity to be used for recovery, use the following identity schema:
+
+```diff
+ {
+   "$id": "https://schemas.ory.sh/presets/kratos/quickstart/email-password/identity.schema.json",
+   "$schema": "http://json-schema.org/draft-07/schema#",
+   "title": "Person",
+   "type": "object",
+   "properties": {
+     "traits": {
+       "type": "object",
+       "properties": {
+         "email": {
+           "type": "string",
+           "format": "email",
+           "ory.sh/kratos": {
+             "credentials": {
+               "password": {
+                 "identifier": true
+               }
+             },
++            "recovery": {
++              "via": "email"
++            }
+           }
+         }
+       }
+       "additionalProperties": false
+     }
+   }
+ }
+```
+
+## Email templates
+
+Ory Identities (Ory Kratos) provides a set of default email templates for the recovery flows.
+
+:::important
+
+In case a user tries to recover an unregistered account, an email is sent out to the email, informing them about the attempted
+recovery.
+
+This prevents account enumeration attacks as explained in this
+[blog post by Troy Hunt](https://www.troyhunt.com/website-enumeration-insanity-how-our-personal-data-is-leaked/).
+
+:::
+
+To provide your own templates, see [Customizing E-Mail Templates](../../emails-sms/custom-email-templates).
+
+## Updating credentials
+
+After the user completes the recovery successfully they receive a privileged session and are redirected to the settings page to
+update their credentials.
+
+Read more about the [privileged sessions](../../session-management/01_overview.mdx#privileged-sessions).
+
+## Invalidate other sessions
+
+In some scenarios it can be useful to revoke all active sessions once an account has been recovered. This forces anyone with
+access to the account to re-authenticate.
```

Comment:
To get that working in Ory Network you'd have to edit the config directly, there's no way to do that through the UI, right? We should explain that. 
Also, needs explanation that it's done through hooks (so you know that even before you look at the config snippet) and it's a good opportunity to our hooks docs.

Reply:
Good point! Done.